### PR TITLE
Table data optimization

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,48 @@
...
+1. Create pull request. Pull requests will then be reviewed and approved/declined.
+
+**Remember to keep commits and pull requests as small as you can.**
+
+## Angular Docs
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.23.
 
-## Development server
+### Development server
```
Comment (kaseyllabelle) [inline README.md]: Can we remove this since we added 'running the application'? Or at least combine the two?

File: scripts/refresh-table-data.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  const loremArr = lorem.split(' ');
+  const descArr = loremArr.slice(0, getRandomInt(4, loremArr.length));
+  return randomSelect(chars) + descArr.join(' ') + '.';
+}
+
+function getAlert() {
+  return randomSelect([true, false]);
+}
+
+
```
Comment (kaseyllabelle) [inline scripts/refresh-table-data.js]: Delete extra white space

File: scripts/refresh-table-data.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    'Not Deployed',
+    'New Updates Not Deployed',
+    'Deploying',
+    'Deployment Error',
+    'Deployed'
+  ]);
+}
+
+function getDescription() {
+  const lorem = 'lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse bibendum quam non purus semper viverra. Aenean tristique rhoncus velit lobortis faucibus';
```
Comment (kaseyllabelle) [inline scripts/refresh-table-data.js]: Can we make Lorem sentence case?

Comment (APJohns) [reply scripts/refresh-table-data.js]: It is sentence case, I am adding an uppercase letter onto the beginning of the string randomly so we can demo sorting by description.

**Pull request merged**